src/Routes.js

```diff
@@ -80,6 +88,28 @@ export const AppRoutes = () => (
           />
         }
       />
+      <Route
+        path="/workloads"
+        element={
+          <WorkloadsList
+            /**
+             * Generate random `key` to force component re-render,
+             * temporary workaround for https://issues.redhat.com/browse/OCPADVISOR-59
+             */ key={Math.random()}
```

Comment:
Can you please check if https://issues.redhat.com/browse/OCPADVISOR-59 is reproducible if we remove this `key` workaround now? If we have enough time, I would vote for investigating how this could be fixed properly (though if it's no longer reproducible, let's forget about it and remove `key` now).

Reply:
The issue is still present. We can investigate it in a separate task, but it shouldn't be a part of the "structure setup PR"